boards/arm/raytac_mdbt53_db_40_nrf5340/doc/index.rst

```diff
@@ -0,0 +1,349 @@
+
+- Implementation Defined Attribution Unit (`IDAU`_) on the application core.
+  The IDAU is implemented with the System Protection Unit and is used to
+  define secure and non-secure memory maps.  By default, all of the memory
+  space  (Flash, SRAM, and peripheral address space) is defined to be secure
+  accessible only.
+- Secure boot.
+
+Programming and Debugging
+*************************
+
+nRF5340 application core supports the Armv8-M Security Extension.
+Applications built for the raytac_mdbt53_db_40_nrf5340_cpuapp board by
+default boot in the Secure state.
+
+nRF5340 network core does not support the Armv8-M Security Extension.
+nRF5340 IDAU may configure bus accesses by the nRF5340 network core
+to have Secure attribute set; the latter allows to build and run
+Secure only applications on the nRF5340 SoC.
+
+Applications for the ``raytac_mdbt53_db_40_nrf5340`` board configuration can be
+built, flashed, and debugged in the usual way. See :ref:`build_an_application` and
+:ref:`application_run` for more details on building and running.
+
+.. note::
+   Flashing and Debugging Zephyr onto the raytac_mdbt53_db_40_nrf5340 board
+   requires an	external J-Link programmer. The programmer is attached to the J1
+   or J9 SWD connector.
+
+Building Secure/Non-Secure Zephyr applications with Arm |reg| TrustZone |reg|
+=============================================================================
+
+Applications on the nRF5340 may contain a Secure and a Non-Secure firmware
+image for the application core. The Secure image can be built using either
+Zephyr or `Trusted Firmware M`_ (TF-M). Non-Secure firmware
+images are always built using Zephyr. The two alternatives are described below.
+
+.. note::
+
+   By default the Secure image for nRF5340 application core is built
+   using TF-M.
+
+
+Building the Secure firmware with TF-M
+--------------------------------------
+
+The process to build the Secure firmware image using TF-M and the Non-Secure
+firmware image using Zephyr requires the following steps:
+
+1. Build the Non-Secure Zephyr application
+   for the application core using ``-DBOARD=raytac_mdbt53_db_40_nrf5340_cpuapp_ns``.
+   To invoke the building of TF-M the Zephyr build system requires the
+   Kconfig option ``BUILD_WITH_TFM`` to be enabled, which is done by
+   default when building Zephyr as a Non-Secure application.
+   The Zephyr build system will perform the following steps automatically:
+
+      * Build the Non-Secure firmware image as a regular Zephyr application
+      * Build a TF-M (secure) firmware image
+      * Merge the output image binaries together
+      * Optionally build a bootloader image (MCUboot)
+
+.. note::
+
+   Depending on the TF-M configuration, an application DTS overlay may be
+   required, to adjust the Non-Secure image Flash and SRAM starting address
+   and sizes.
+
+2. Build the application firmware for the network core using
+   ``-DBOARD=raytac_mdbt53_db_40_nrf5340_cpunet``.
+
```

Comment:
.. note::
   Trusted Firmware-M (TF-M) and building the ``ns`` target is not supported for this board.

Reply:
Could you also add this note?
The other boards all have this note.

Reply:
I have added this note.